### ISSUE TITLE
Custom Validation is not triggered for empty Array

### ISSUE THREAD

**Issue by sabbir-hossain** [mikeerickson/validatorjs#332]
I have the following Custom Validation Rules:

```
const Validator = require("validatorjs");

Validator.register(
  "NotEmptyArray",
  (value, requirement, attribute) => {
    return Array.isArray(value) && value[0] !== "";
  },
  "The :attribute cannot be empty array"
);

```

And when I'm trying to validate following data, it is not working as expected.

```
const value = new Validator(
  { label: "Question 1", widgetId: [""] },
  { label: "string", widgetId: "NotEmptyArray" }
);
```

it always return `true`

**Comment by shuhailshuvo** [comment]
@shsaucorp,
You don't need to register custom rules in your case.
Just validate `widgetId` by `required|min:1`

you can check this [repl](https://repl.it/repls/UtilizedRigidStatistics)

**Comment by sabbir-hossain** [comment]
@shuhailshuvo thanks for your help

**Comment by dcworldwide** [comment]
Same requirement and issue...however I need a custom function to do additional array filter checks. I can't get by with min:x.

This should be an open issue
